src/rgw/rgw_lua_background.h

```diff
@@ -0,0 +1,51 @@
+  #pragma once
+  #include <lua.hpp>
+  #include "common/dout.h"
+  #include "rgw_common.h"
+  
+  namespace rgw::lua {
+  
+  class Background {
+      protected:
+          std::unordered_map<std::string, int> rgw_map;
+          std::string rgw_script;
+          std::mutex m_mutex;
+          bool stopped;
+  	  int set_off;
+          const DoutPrefixProvider* dpp;
+          rgw::sal::Store* store;
+          std::thread runner;
+          CephContext* cct;
+          lua_State* L;
+      public:
+          Background(
+	      const DoutPrefixProvider* dpp,
+	      rgw::sal::Store* const store,
+	      CephContext* cct,
+	      lua_State* L)
+	  : stopped(false),
```

Comment:
better off initializing member variables in-class for better readability.

Reply:
Could you please clarify?
Thank you for the CR Kefu.
btw, this draft PR is still WIP.

Reply:
when you initialize a data member with some const value, you can do that at the point of declaration. (e.g. `bool stopped = true`) instead of the init list